### PR TITLE
feat(uib): Add Interact with Bonita page

### PR DESCRIPTION
**Pull request by jeromecambon**
Covers [BPM-128](https://bonitasoft.atlassian.net/browse/BPM-128)

[BPM-128]: https://bonitasoft.atlassian.net/browse/BPM-128?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/applications/pages/download-and-launch.adoc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 = Download and launch Documentation
+:page-aliases: ROOT:download-and-launch.adoc
```
Comment (benjaminParisel) [inline modules/applications/pages/download-and-launch.adoc]: the :page-aliases: must be here in only 2 scenarios: When the referenced page is moved, or when the referenced page is deleted
```suggestion
```

File: modules/applications/pages/interact-with-your-bonita-process.adoc [hunk 2]
```diff
@@ -1,7 +1,41 @@
 = Interact with your Bonita process
-:description:
+:page-aliases: ROOT:interact-with-your-bonita-process.adoc
```
Comment (benjaminParisel) [inline modules/applications/pages/interact-with-your-bonita-process.adoc]:
```suggestion
```

File: modules/applications/pages/bonita-ui-builder.adoc [hunk 3]
```diff
@@ -13,6 +13,10 @@ They are not anymore standalone elements (such as pages and forms) in a process
 xref:ROOT:builder-declare-interface-in-bonita.adoc[[.card-title]#Declare the interface in Bonita# [.card-body.card-content-overflow]#pass:q[Steps to declare your interface and display it in Bonita]#]
 --
 
+[.card.card-index]
+--
+xref:ROOT:interact-with-your-bonita-process.adoc[[.card-title]#Interact with your Bonita process# [.card-body.card-content-overflow]#pass:q[Steps to interact with your Bonita process]#]
```
Comment (benjaminParisel) [inline modules/applications/pages/bonita-ui-builder.adoc]: ❌ The link is not correct if you remove the page-aliases, you can found example here [bonitasoft.github.io/bonita-documentation-theme/links.html](https://bonitasoft.github.io/bonita-documentation-theme/links.html)

File: modules/applications/pages/interact-with-your-bonita-process.adoc [hunk 4]
```diff
@@ -1,7 +1,41 @@
...
 ====
 For Subscription editions only.
 ====
+
+== Create a Bonita datasource
+From a page in edit mode, click the `Data` icon from the left panel.
+Then, you get the list of the provided datasources.
+
+From the `APIs` section, click on the `Bonita API`. By default, it will connect to the Bonita runtime running on the local machine. No need to provide any credential information,
+since they are already provided at the xref:ROOT:download-and-launch.adoc[Download and launch] step.
```
Comment (benjaminParisel) [inline modules/applications/pages/interact-with-your-bonita-process.adoc]: Update link too

**Review by benjaminParisel — CHANGES_REQUESTED** [review]
Small updates needed, more Antora Issue than content issue

File: modules/applications/pages/download-and-launch.adoc (hunk 1 above)
Comment (jeromecambon) [reply modules/applications/pages/download-and-launch.adoc]: ha yes that's true...

**Review by abirembaut — APPROVED** [review]
LGTM

**Review by benjaminParisel — APPROVED** [review]

**Pull request merged**